cinema/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.CharField(max_length=255)
+    duration = models.IntegerField()
+
+    class Meta:
+        verbose_name_plural = "cinemas"
```

Comment:
The `verbose_name_plural` is set to 'cinemas', which seems incorrect for the `Movie` model. It should likely be 'movies' to accurately reflect the model's purpose.